### ISSUE TITLE
deprecation with symfony 4.2.1

### ISSUE THREAD

**Issue by behalp** [kevinpapst/AdminLTEBundle#41]
After update to symfony 4.2.1 with latest AdminLTEBundle 2.1.3 I get following deprecated:

`User Deprecated: The "KevinPapst\AdminLTEBundle\Controller\EmitterController" class extends "Symfony\Bundle\FrameworkBundle\Controller\Controller" that is deprecated since Symfony 4.2, use {@see AbstractController} instead. *.`

and

```
A tree builder without a root node is deprecated since Symfony 4.2 and will not be supported anymore in 5.0.
Show context    Hide trace
{▼
  E:\dokumenty\projects\symfony4_hrbitov\vendor\symfony\config\Definition\Builder\TreeBuilder.php:30 {▶}
  E:\dokumenty\projects\symfony4_hrbitov\vendor\kevinpapst\adminlte-bundle\DependencyInjection\Configuration.php:27 {▼
    › {
    ›     $treeBuilder = new TreeBuilder();
    ›     $rootNode = $treeBuilder->root('admin_lte');
  }
}
```

**Comment by behalp** [comment]
and after clear of cache I get deprecated about transchoice


```
Twig Filter "transchoice" is deprecated since version 4.2. Use "trans" with parameter "%count%" instead in E:\dokumenty\projects\symfony4_hrbitov\vendor\kevinpapst\adminlte-bundle\Resources\views\Navbar\messages.html.twig at line 8.

Twig Filter "transchoice" is deprecated since version 4.2. Use "trans" with parameter "%count%" instead in E:\dokumenty\projects\symfony4_hrbitov\vendor\kevinpapst\adminlte-bundle\Resources\views\Navbar\notifications.html.twig at line 7

 Twig Filter "transchoice" is deprecated since version 4.2. Use "trans" with parameter "%count%" instead in E:\dokumenty\projects\symfony4_hrbitov\vendor\kevinpapst\adminlte-bundle\Resources\views\Navbar\tasks.html.twig at line 7.
```

**Comment by kevinpapst** [comment]
Thanks for the info, I will have a look soon.

**Comment by kevinpapst** [comment]
To everyone interested: I cannot fix these deprecations for now, as that would be a BC break.

**Comment by jkabat** [comment]
I think that both of the issues and much more could be fixed without BC break - can we elaborate?

**Comment by kevinpapst** [comment]
@jkabat Unfortunately I forgot what was the reason for my believe, should have written it down ... 
But I pushed my branch and you could have a look and see if we can fix it #52

**Comment by jkabat** [comment]
thanks, Ill take a look tomorrow.

**Comment by kevinpapst** [comment]
@behalp could you give the PR #52 a try ?

**Comment by kevinpapst** [comment]
Fixed with 2.4
